arbitrum-docs/how-arbitrum-works/10-l1-to-l2-messaging.mdx

```diff
@@ -1,147 +1,437 @@
+- You need an operation on a child chain that explicitly requires EOA authorization (e.g., a withdrawal).
 
-[enqueue_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/block_processor.go#L245
-[exhaust_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/block_processor.go#L135
-[discard_link]: https://github.com/OffchainLabs/nitro/blob/fa36a0f138b8a7e684194f9840315d80c390f324/arbos/retryables/retryable.go#L262
-[renew_link]: https://github.com/OffchainLabs/nitro-contracts/blob/a68783436b5105a64f54efe5fbd55174704a7618/src/precompiles/ArbRetryableTx.sol#L41
+#### How signed messages work
 
-![400px-img](../assets/manual-redemption-of-the-ticket.png)
+When submitting through the delayed inbox, a child chain transaction signature gets included in the message's calldata. Because it matches the EOA's signature, the child chain can safely treat the signer's address as the sender.
+Example use case: [Withdraw Ether tutorial](https://github.com/OffchainLabs/arbitrum-tutorials/blob/a1c3f64a5abdd0f0e728cb94d4ecc2700eab7579/packages/delayedInbox-l2msg/scripts/withdrawFunds.js#L61-L65)
 
-:::caution Avoid Losing Funds!
+#### Delayed inbox methods for signed messages
 
-If a ticket expires after 7 days without being redeemed or re-scheduled to a future date, any message and value (other than the escrowed `callvalue`) it carries could be lost without possibility of being recovered.
+There are two primary methods for sending signed messages:
 
+1. `sendL2Message`
+
+- Only an EOA with no deployed code can call this ("codeless origin")
+- The signed transaction is retrieved directly from calldata, so emitting a large event log is unnecessary
+- Offers lower gas costs (cheaper)
+
+```solidity
+function sendL2Message(
+    bytes calldata messageData
+) external whenNotPaused onlyAllowed returns (uint256)
+```
+
+2. `sendL2MessageFromOrigin`
+
+- More flexible
+- It can be called by either an EOA or a contract
+- The full signed transaction data is emitted in an event log so that nodes can reconstruct the transaction without replaying it
+
+```solidity
+function sendL2MessageFromOrigin(
+    bytes calldata messageData
+) external whenNotPaused onlyAllowed returns (uint256);
+```
+
+### Unsigned messages
+
+Unsigned messages allow a Parent chain sender to specify transaction parameters without an EOA signature. Because there is no signature, **the sender's address must be aliased on the child chain** (see the [Address aliasing](#address-aliasing) section for the rationale.). The delayed inbox provides four main methods for unsigned messages, divided based on whether the sender is an EOA or a contract and whether it includes parent chain funds:
+
+1. **Unsigned from EOAs**: These methods incorporate a none for replay protection, similar to standard EOA-based transaction on Ethereum.
```

Comment:
```suggestion
Unsigned messages allow a Parent chain sender to specify transaction parameters without an EOA signature. Because there is no signature, **the sender's address must be aliased on the child chain** (see the [Address aliasing](#address-aliasing) section for the rationale.). The delayed inbox provides four main methods for unsigned messages, divided based on whether the sender is an EOA or a contract and whether it includes parent chain funds:

1. **Unsigned from EOAs**: These methods incorporate a none-for replay protection, similar to standard EOA-based Ethereum transactions.
```